gazebo_ros/include/gazebo_ros/conversions/builtin_interfaces.hpp

```diff
@@ -27,7 +27,7 @@ namespace gazebo_ros
 /// \brief Specialized conversion from an Gazebo Time to a ROS Time message.
 /// \param[in] in Gazebo Time to convert.
 /// \return A ROS Time message with the same value as in
-template<>
+template<> inline
```

Comment:
Looks like our linters prefer the `inline` keyword on the next line, e.g. 
```suggestion
template<>
inline
```

Same for all other instances.

Reply:
I went ahead and pushed a commit to address this ([7a79eff](https://github.com/ros-simulation/gazebo_ros_pkgs/pull/1367/commits/7a79eff53a95fa43209b8cf3f31cdc5c6a485e82)).